apps/web/pages/codercamp.tsx

```diff
@@ -0,0 +1,117 @@
+import { Footer, Header } from "@coderdojobraga/ui";
+import { MessageFilled, ScheduleOutlined } from "@ant-design/icons";
+import Image from "next/image";
+
+const CoderCamp = () => (
+  <>
+    <Header landing={true} />
+
+    <div className="container mx-auto mt-12">
+      <h1
+        className="m-4 text-center text-4xl font-bold sm:text-5xl"
+        style={{ fontFamily: "Bankai", fontSize: "5rem", lineHeight: "5rem" }}
+      >
+        Coder Camp
```

Comment:
I would refer specifically the year of the event
```suggestion
        Coder Camp '23
```